lib/facter/rabbitmq_plugins_dirs.rb

```diff
@@ -2,8 +2,12 @@
   setcode do
     if Facter::Util::Resolution.which('rabbitmqctl')
       rabbitmq_pluginsdirs_env = Facter::Core::Execution.execute("rabbitmqctl eval 'application:get_env(rabbit, plugins_dir).'")
-      rabbitmq_plugins_dirs = %r{^\{ok\,\"(\/.+\/\w+)}.match(rabbitmq_pluginsdirs_env)[1]
-      rabbitmq_plugins_dirs.split(':')
+      rabbitmq_plugins_dirs_match = %r{^\{ok\,\"(\/.+\/\w+)}.match(rabbitmq_pluginsdirs_env)
+      if rabbitmq_plugins_dirs_match then
+        rabbitmq_plugins_dirs_match[1].split(':')
+      else
```

Comment:
Sorry, seems like I like this style but rubocop doesn’t (see Travis failures). I think you can just return `nil` if the condition doesn’t match with the if.

Reply:
Yea if i follow the rubocop recommendation and just remove the `else` branch, it should work fine, the `if` should return `nil` in that case anyway.